platform/platform-configuration-listener/pom.xml

```diff
@@ -29,6 +29,10 @@
     </description>
     <packaging>bundle</packaging>
 
+    <properties>
```

Comment:
I tried changing the pom as suggested, but it didn't seem to build anymore.  Leaving for now.